strawberry/schema/config.py

```diff
@@ -21,5 +24,8 @@ def __post_init__(
         if auto_camel_case is not None:
             self.name_converter.auto_camel_case = auto_camel_case
 
+        if not issubclass(self.info_class, Info):
+            raise ValueError("`info_class` must be a subclass of strawberry.types.Info")
```

Comment:
can we add a `# pragma: no cover`? here? 😊

Reply:
Sure!